src/pageScanner/checks/aria-hidden-valid-usage.js

```
@@ -0,0 +1,122 @@
+				return true;
+			}
+		}
+
+		// Check for valid classes
+		if ( node.classList.contains( 'wp-block-spacer' ) ) {
+			return true;
+		}
+
+		// Check for valid roles
+		const role = node.getAttribute( 'role' );
+		if ( role && role.split( /\s+/ ).includes( 'presentation' ) ) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)